csharp/ParquetSharp.csproj

```diff
@@ -75,6 +75,7 @@
 
   <ItemGroup>
     <PackageReference Include="Apache.Arrow" Version="13.0.0" />
+    <PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.1" />
```

Comment:
Can you move this down inside the conditional block (`<ItemGroup Condition="'$(TargetFramework)' == 'net471' or '$(TargetFramework)' == 'netstandard2.0'">`)